tests/unit/test_cli.py

```
@@ -1,20 +1,38 @@
 from click.testing import CliRunner
+import os
+import unittest
+from unittest import mock
 
 from secator.cli import cli
-
-import unittest
+from secator.definitions import VERSION
+from secator.installer import InstallerStatus, PackageInstaller, SourceInstaller
```

Ruff (0.11.9)

`secator.installer.PackageInstaller` imported but unused

Remove unused import

(F401)